src/content/docs/telemetry-data-platform/understand-data/metric-data/query-metric-data-type.mdx

```diff
@@ -235,6 +235,35 @@ If used in a query, `myNeatProcess.%.duration` will return results for all three
   </Collapser>
 </CollapserGroup>
 
+## Return results for invidual fields using `getField()`
+
+Metrics have multiple fields associated with them, and each field contains its own unique values. If you would like to extract one of those values using a query, you can do that with `getField()`. 
+
+For example, if you want to use a single value within a metric to do a comparison in a `WHERE` clause, you can do that with `getField(metricName, field)` or `metricName[field]`.
```

Comment:
You've used the expression "you can do that with" in the previous sentence, I'd rewrite this.